svg.panzoom.js

```diff
@@ -43,20 +42,22 @@ SVG.extend(SVG.Doc, SVG.Nested, {
     var pinchZoomStop = function(ev) {
       ev.preventDefault()
       zoomInProgress = false
-      
+
       SVG.off(document,'touchmove', pinchZoom)
       SVG.off(document,'touchend', pinchZoomStop)
       this.on('touchstart', pinchZoomStart)
```

Comment:
As you can see the document listeners are only for touchmove and touchend. Not for touchstart.
This is generally needed to make sure, that you also can continue panning when you out of the borders of the orginally box. Restricting the users pan to the box only feels clunky and is not what the user would expect.

So thats why we have this. The handlers on document will never interfer with a second map because they are bound on touchdown so we are sure the user selected an item to pan